core/integration/features/debug.feature

```diff
@@ -0,0 +1,112 @@
+      | id        | quote name | asset | risk model                | margin calculator   | auction duration | fees         | price monitoring | data source config | linear slippage factor | quadratic slippage factor | decimal places | position decimal places | parent market id | insurance pool fraction | successor auction | sla params |
+      | ETH/DEC19 | ETH        | USD   | lognormal-risk-model-fish | margin-calculator-1 | 1                | default-none | default-none     | ethDec19Oracle     | 0.1                    | 0                         | 0              | 0                       |                  |                         |                   | SLA        |
+    When the parties place the following orders:
+      | party   | market id | side | volume | price | resulting trades | type       | tif     |
+      | trader1 | ETH/DEC19 | buy  | 1 | 1000 | 0 | TYPE_LIMIT | TIF_GTC |
+      | trader2 | ETH/DEC19 | sell | 1 | 1000 | 0 | TYPE_LIMIT | TIF_GTC |
+    #When the network moves ahead "2" blocks
+    Then the market data for the market "ETH/DEC19" should be:
+      | trading mode                 | auction trigger         | target stake | supplied stake | open interest |
+      | TRADING_MODE_OPENING_AUCTION | AUCTION_TRIGGER_OPENING | 4878         | 0              | 0             |
+
+    When the parties submit the following liquidity provision:
+      | id  | party   | market id | commitment amount | fee | lp type    |
+      | lp1 | lpprov1 | ETH/DEC19 | 9000              | 0.1 | submission |
+
+    And the market data for the market "ETH/DEC19" should be:
+      | mark price | trading mode                 | target stake | supplied stake | open interest |
+      | 0          | TRADING_MODE_OPENING_AUCTION | 4878         | 9000           | 0             |
+    Then the opening auction period ends for market "ETH/DEC19"
+    And the network moves ahead "20" blocks
+    #And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/DEC19"
+    And the market data for the market "ETH/DEC19" should be:
+      | mark price | trading mode            | auction trigger             | target stake | supplied stake | open interest |
+      | 1000       | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED | 4878         | 3140           | 1             |
+
+    Then the following transfers should happen:
+      | from    | to     | from account      | to account             | market id | amount | asset |
+      | lpprov1 | market | ACCOUNT_TYPE_BOND | ACCOUNT_TYPE_INSURANCE | ETH/DEC19 | 1710   | USD   |
+      | lpprov1 | market | ACCOUNT_TYPE_BOND | ACCOUNT_TYPE_INSURANCE | ETH/DEC19 | 1385   | USD   |
+      | lpprov1 | market | ACCOUNT_TYPE_BOND | ACCOUNT_TYPE_INSURANCE | ETH/DEC19 | 1121   | USD   |
+      | lpprov1 | market | ACCOUNT_TYPE_BOND | ACCOUNT_TYPE_INSURANCE | ETH/DEC19 | 908    | USD   |
+      | lpprov1 | market | ACCOUNT_TYPE_BOND | ACCOUNT_TYPE_INSURANCE | ETH/DEC19 | 736    | USD   |
+
+
+# @SLADebug
```

Comment:
This scenario has been commented out, do we still need it?

Reply:
This scenario was a bit of a copy-pasta to try and work out what was going on. It's still a setup that mimics some of the system test steps, so I figured I'd keep the first part, but I can delete the parts we don't need